R/html-tweak.R

```diff
@@ -251,38 +251,46 @@ tweak_homepage_html <- function(html, strip_header = FALSE) {
 
 # Mutates `html`, removing the badge container
 badges_extract <- function(html) {
-  # First try specially named div;
-  x <- xml2::xml_find_first(html, "//div[@id='badges']")
+  # First try specially named element;
+  x <- xml2::xml_find_first(html, "//*[@id='badges']")
+  force <- TRUE
```

Comment:
I think `strict <- FALSE` might be a more informative name? (i.e. strict = only badges no other html)

Reply:
makes sense!